src/org/usfirst/frc/team166/robot/subsystems/Drive.java

```diff
@@ -137,6 +141,10 @@ public double angleError() {
 			return 360 - gyro.getAngle();
 	}
 
+	public double motorCompAngle(double angleError) {
+		return 1.65321 * Math.log10(angleError);
```

Comment:
Where do these numbers come from? Can you verify its accuracy or explain its reasoning in a comment?